pandera/schema_statistics.py

```diff
@@ -158,7 +159,12 @@ def parse_checks(checks) -> Union[Dict[str, Any], None]:
     check_statistics = {}
     _check_memo = {}
     for check in checks:
-        check_statistics[check.name] = check.statistics
+        if check not in Check:
+            warnings.warn("only registered checks may be converted to statistics. "
```

Comment:
"only" should start with capital "O" :)

Reply:
We have a different convention, because this is rendered as `UserWarning: only ...`. But I capitalized the `O` for consistency.